app/src/main/kotlin/at/bitfire/davdroid/repository/AccountRepository.kt

```diff
@@ -180,6 +190,16 @@ class AccountRepository @Inject constructor(
         }
     }
 
+    /**
+     * Get user data stored in account.
+     */
+    fun getUserData(account: Account, key: String) = accountManager.getUserData(account, key)
```

Comment:
Such a method could be here¹, but it's used to access _address book accounts_, so in this context it doesn't belong here (rather to `LocalAddressBook` or a similar class).

¹ Side note: I think a generic "get data" method may be too generic for a repository. The underlying data source may contain such a method, but at latest the repository should then provide specific methods like "get URL" etc.

Reply:
Right